docs/tutorials/license-server-auth.md

```diff
@@ -89,6 +89,8 @@ We can use a request filter to modify the URL and add the required parameter:
     // Only add headers to license requests:
     if (type == shaka.net.NetworkingEngine.RequestType.LICENSE) {
       // This is the specific parameter name and value the server wants:
+      // Note that there would be multiple uris for lisence server.
```

Comment:
Small spelling error: "license".  Also, it would be more accurate to say something like

"All network requests can have multiple URIs (for fallback), and therefore this is an array.  But there should only be one license server URI."